src/lib.rs

```diff
@@ -2953,6 +2974,40 @@ impl Connection {
                 // Once the handshake is confirmed, we can drop Handshake keys.
                 self.drop_epoch_state(packet::EPOCH_HANDSHAKE);
             },
+
+            frame::Frame::AckFrequency {
+                sequence_number,
+                packet_tolerance,
+                update_max_ack_delay,
+                ignore_order,
+            } => {
```

Comment:
QUIC extension frames require opt in, so we should blow up if the frames is received in a connection that does not support the extension. 

This is actually going to be a common pattern, so we can probably write a common extension checker and have a single unit test for extensions.

Reply:
We can detect is using min_ack_delay value. I added a check to return InvalidFrame is min_ack_delay is not set. Supporting an extension need to be controlled via config APIs. I added Config.ext_set_min_ack_delay() for this extension -- by default it's not enabled. But it will be good to have some common stuff for extensions.